ivre/webutils.py

```diff
@@ -245,6 +245,10 @@ def add_unused(neg, param, value):
                 limit = min(limit, config.WEB_MAXRESULTS)
         elif param == "archives":
             archive = not neg
+        elif param == "id":
+            list_id = [str(oid) for oid in value.replace('-', ',').split(',')]
```

Comment:
Why do you need the `str()` constructor here? Wouldn't `oids = value.replace('-', ',').split(',')` be enough?

And if so, defining a variable to use it once makes little sense, write:

```Python
    flt = db.nmap.flt_and(flt, db.nmap.searchobjectid(
        value.replace('-', ',').split(','), neg=neg))
```